gradle/libs.versions.toml

```diff
@@ -167,7 +166,6 @@ validation-api = { module = "jakarta.validation:jakarta.validation-api"}
 annotation-api = { module = "jakarta.annotation:jakarta.annotation-api"}
 
 # code quality
-diktat-gradle-plugin = { module = "org.cqfn.diktat:diktat-gradle-plugin", version.ref = "diktat" }
```

Comment:
I think we can leave it here, but it can point to new group id already (just not need to use as dependency for now)